docker/test.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-APP=core
+APP=${1:-core}
+VERBOSITY=${2:-0}
```

Comment:
Ooh, it's bash magic at its worst.  `${1:-foo}` means *give me $1 unless it's not initialized, in which case give me "foo"*.